werkzeug/exceptions.py

```diff
@@ -672,7 +672,27 @@ def __call__(self, code, *args, **kwargs):
             raise LookupError('no exception for %r' % code)
         raise self.mapping[code](*args, **kwargs)
 
-abort = Aborter()
+
+def abort(status, *args, **kwargs):
```

Comment:
@Marx314 I think it's not a good idea to change it back, since this would break people's code twice.